docs/changes/20171027-organization-users.md

```diff
@@ -0,0 +1,5 @@
+## Organization Users
+October 27, 2017
+
+Organization resource requests will no longer return a users collection. To
+get users for an organization, see the [Users](/api/users) documentation.
```

Comment:
oh, i figured it would be GET /organizations/{id} ?

Reply:
Not according to the flowdock-web. Organization endpoint no longer returns users ever. Which probably isn't a problem?

Reply:
/organizations/:id/users is documented in users.md

Flow users is also documented there. We could repeat it here, but I put a link instead.

Reply:
I suppose we could return for /organizations/:id

Thoughts?